CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ Adds support to API300 HPE Synergy Image Streamer resources:
   - image_streamer_artifact_bundle
   - image_streamer_os_build_plan
 
+Enhancements:
+  - connection_template now supports `EthernetNetwork`, `FCoENetwork`, `FCNetwork` and `NetworkSet`
```

Comment:
@jsmartt Thanks, I'll definitely need some help with unit tests. What exactly needs to be done? Is it something in here? https://github.com/HewlettPackard/oneview-chef/tree/master/spec I'm sorry for these questions :)